e2e/cypress/integration/enterprise/ldap/ldap_test_setup.txt

```diff
@@ -0,0 +1,18 @@
+1. Provision the MM app in LDAP Provider following the steps described here: https://docs.mattermost.com/deployment/sso-ldap.html
```

Comment:
should this be moved to https://github.com/mattermost/platform-private/, similar to https://github.com/mattermost/platform-private/blob/master/config/saml-okta-setup.txt ?

Reply:
Possibly, the LDAP didn't have any other files, such as certificates. So I didn't know if it made sense to just put that single file in `platform-private`. @srkgupta Let me know where you would like this txt file.

Reply:
If the file is fine to be in public and does not contain confidential information, then I'm good having it here.  Otherwise, it makes sense to move as suggested.  Like for example, is it okay for public to know about `ldap.e2e.dev.spinmint.com` or the AWS server (to be setup in the future)?

Reply:
Moved